phabfive/maniphest.py

```diff
@@ -14,14 +17,83 @@
 import yaml
 from jinja2 import Template
 
-
 log = logging.getLogger(__name__)
 
-
 class Maniphest(Phabfive):
     def __init__(self):
         super(Maniphest, self).__init__()
 
+    def alex_search(self, created_after=None, updated_after=None, project=None):
+        """
+        Search for Phabricator Maniphest tasks with given parameters.
+
+        Parameters
+        ----------
+        created_after (int, optional): Number of days ago the task was created.
+        updated_after (int, optional): Number of days ago the task was updated.
+        project (str, optional): Project name.
```

Comment:
Since all arguments are optional, what happens if you call `phabfive maniphest search` without any arguments? It should probably avoid returning all tasks from the Phabricator instance and instead tell the user that some kind of narrowing down of the search query should be preferred, or add some kind of `--all-tasks` argument that actually returns everything.

I am thinking as an end-user here. What I would expect from a UX perspective.

Reply:
--all-tasks argument is not required since the function already returns all tasks for the given project.